homeassistant/components/slack/__init__.py

```diff
@@ -1,2 +1,68 @@
-"""The slack component."""
-DOMAIN = "slack"
+"""The slack integration."""
+import logging
+
+from aiohttp.client_exceptions import ClientError
+from slack import WebClient
+from slack.errors import SlackApiError
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_PLATFORM, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import aiohttp_client, discovery
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DATA_CLIENT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.NOTIFY]
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Slack component."""
+    # Iterate all entries for notify to only get Slack
+    if Platform.NOTIFY in config:
```

Comment:
You could reverse the condition and return true right away to be able to save some indent but may not be worth it since you still have to return True below